recipes/gclib/meta.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  name: gclib
+  version: "907a9d"
+
+build:
+    number: 0
+    skip: False
+
+source:
+  git_url: https://github.com/gpertea/gclib.git
```

Comment:
@davebx this is correct. At bioconda we do backup all tarballs on Galaxy depot, so we have a mirror of everything. This backup is much easier to do with real tarballs and not git urls. Hope this makes sense.